cpp/util/etcd.cc

```diff
@@ -46,6 +46,9 @@ DEFINE_bool(etcd_consistent, true, "Add consistent=true param to all requests. "
             "Do not turn this off unless you *know* what you're doing.");
 DEFINE_bool(etcd_quorum, true, "Add quorum=true param to all requests. "
             "Do not turn this off unless you *know* what you're doing.");
+DEFINE_int32(etcd_connection_timeout_seconds, 10,
```

Comment:
We might have had a split of "secs" vs "seconds" in our flags (look at line 43). :wink: